GetIntoTeachingApiTests/Services/StoreTests.cs

```diff
@@ -96,26 +96,54 @@ public async void SyncAsync_UpdatesExistingTeachingEvents()
         }
 
         [Fact]
-        public async void SyncAsync_DeletesOrphanedTeachingEventsAndBuildings()
+        public async void SyncAsync_DeletesOrphanedTeachingEvents()
         {
-            await SeedMockTeachingEventsAsync();
+            await SeedMockTeachingEventsAndBuildingsAsync();
             var teachingEvents = MockTeachingEvents().ToList();
             _mockCrm.Setup(m => m.GetTeachingEvents(It.Is<DateTime>(d => CheckGetTeachingEventsAfterDate(d)))).Returns(teachingEvents.GetRange(0, 1));
 
             await _store.SyncAsync();
 
             DbContext.TeachingEvents.Should().BeEquivalentTo(teachingEvents.GetRange(0, 1));
-            DbContext.TeachingEventBuildings.Should()

```

Comment:
Sorry I maybe said to remove this in a previous comment (can't remember) but it's actually testing that when teaching events are saved they get correctly connected to the buildings (its been flagged as missing coverage by SonarCloud)